src/KDSoapClient/KDSoapJob.h

```diff
@@ -98,7 +110,7 @@ class KDSOAP_EXPORT KDSoapJob : public QObject
      * Returns the reply headers received from the SOAP server once the request was completed.
      * Only valid once the request is completed and finished() was emitted.
      */
-    KDSoapHeaders returnHeaders() const;
+    KDSoapHeaders replyHeaders() const;
```

Comment:
You can't rename a public method, that's source incompatible.

If you want to rename this, keep the old method, mark it as deprecated (like Qt does), add the new method next to it (with "\since 1.8").

(Note I think this name initially comes from KDSoapPendingCall, so initially I was going to say "it should remain consistent", but actually... not really. For one call, there's a return value, while here it's about reply to the request, OK.)

Reply:
Normally I would agree about not renaming public methods ... but the implementation was missing. If you actually tried to use "returnHeaders()" you would get a linker error.

Reply:
LOL that's interesting. Whoops, looks like I went a bit fast on that day. Then I fully agree with this change :)